### PR TITLE
Added utility code for LinkedList Node and BST Node

### PR DESCRIPTION
**Pull request by araut1**
Added utility code for LinkedList Node and BST Node, so that solving algorithm problems related to these Data structures is easy.

Let me know of any issues. Happy Building!

### CODE REVIEW

File: ListNode.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+public class ListNode {
```
Comment (anku580) [inline ListNode.java]: Can you please add a folder name `Link List` and then add the ListNode to it.

File: TreeNode.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+public class TreeNode {
```
Comment (anku580) [inline TreeNode.java]: Can you please add a folder name Trees and then add the TreeNode to it.

**Review by anku580 — CHANGES_REQUESTED** [review]
Please address the comments which I've added. After that I will merge the PR.

**Comment by araut1** [comment]
Thanks for reviewing. I have made the changes. Let me know if any issues.

**Review by anku580 — APPROVED** [review]
Hey, thank you for contributing. 

All the best :clap:

**Pull request merged**